ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
```

Comment:
Is this really an ipython notebook or a markdown? Perhaps it will be better to have a single ipython notebook with the 2 files.